README.rst

```diff
@@ -361,7 +361,7 @@ or other unixes.
 
 * dpkg-dev
 * build-essential
-* python3.7-dev and libpython3.7-dev  # use appropriate Python version here
+* python3.9-dev and libpython3.9-dev  # use appropriate Python version here
```

Comment:
Might be even better just to specify `python3-dev` instead?  That will always pull in the current version.  Also, I think libpythonX.Y-dev is unnecessary.  It will get pulled in by python3-dev.